code/src/App.js

```diff
@@ -1,12 +1,15 @@
 import React from 'react';
 import data from './data.json';
-
-console.log(data);
+import { Album } from './components/Album'
 
 export const App = () => {
   return (
-    <div>
-      Find me in src/app.js!
-    </div>
-  );
-}
+    <>
```

Comment:
I see that you used the <> tag! We only learnt about this a week later in the coding session - why do you consider this writing as better than <div> ?